compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -428,7 +428,7 @@ class ReplDriver(settings: Array[String],
   /** Like ConsoleReporter, but without file paths or real -Xprompt'ing */
   private object ReplConsoleReporter extends ConsoleReporter(
     reader = null, // this short-circuits the -Xprompt display from waiting for an input
-    writer = new PrintWriter(out, /* autoFlush = */ true), // write to out, not Console.err
+    writer = new PrintWriter(new BufferedWriter(new OutputStreamWriter(out, StandardCharsets.UTF_8)), /* autoFlush = */ true), // write to out, not Console.err
```

Comment:
Is the BufferedWriter useful for something here?

Reply:
I was wondering the same, but it already took me n hours today to figure out what was actually different (need to toInt chars to get real differences) and how it possibly broke, that I'm done.  This combination comes from what the constructor in PrintWriter does when it's given a PrintStream (out).

Reply:
I guess it doesn't matter since there's autoFlush=true, but I'll note that the repl reporter in scala 2 doesn't use BufferedWriter.